content/03.methods.md

```diff
@@ -185,3 +194,4 @@ We retained articles that were included in the PMCOA corpus and performed a toke
 As above, we generated document embeddings for every obtained preprint and published article.
 We projected these preprint embeddings onto our publication landscape to visually observe the dispersion of this subset.
 Finally, we performed a time analysis that paralleled our approach for the full set of preprint-publication pairs to examine relationships between linguistic changes and the time to publication.   
+The only difference is that we filtered out preprint-publication pairs within our bioRxiv snapshot that had a publication time longer than the slowest publication pair in [@doi:10.1101/2021.02.20.432090].
```

Comment:
This bit needs to now be updated as a result of our discussion: namely the primary result remains the same but this becomes an additional analysis.